jokers/burnt.lua

```diff
@@ -0,0 +1,52 @@
+SMODS.Enhancement({
+    key = "burnt",
+    atlas = "folly_enhancers",
+    pos = { x = 0, y = 0 },
+    min = 0.5,
+    max = 1.5,
```

Comment:
```suggestion
 config = {
        extra = {
            min = 0.5,
            max = 1.5,
        },
    },
```
You should put them in config.
maybe change names too? `x_mult_min`?